htmresearch/encoders/cio_encoder.py

```diff
@@ -135,11 +138,11 @@ def __setstate__(self, state):
     self.__dict__ = state
 
 
-  def _setDimensions(self, retina, scalingFactor):
+  def _setDimensions(self, scalingFactor):
     if scalingFactor <= 0 or scalingFactor > 1:
       raise ValueError("Retina can only be scaled by values between 0 and 1.")
 
-    retinaDim = RETINA_SIZES[retina]["width"]
+    retinaDim = RETINA_SIZES[self.retina]["width"]
```

Comment:
if self.retina is only ever used here (called when initialized) then do we really need a self.retina member variable?